core/services/relay/evm/chain_reader.go

```diff
@@ -173,14 +173,26 @@ func (cr *chainReader) Unbind(ctx context.Context, bindings []commontypes.BoundC
 func (cr *chainReader) GetLatestValue(ctx context.Context, readName string, confidenceLevel primitives.ConfidenceLevel, params any, returnVal any) error {
 	binding, address, err := cr.bindings.GetReader(readName)
 	if err != nil {
+		cr.lggr.Debug(err.Error())
```

Comment:
Is there more to say here?
```suggestion
		cr.lggr.Debugw("message","err",err.Error())
```

Reply:
Maybe this logging should be the responsibility of the caller. I did it here because this was the request as I understood it. My primary goal was to put more structured information in the errors themselves and ensure they are wrapped with the correct types for the loop interface.